lib/locales/en.yml

```diff
@@ -16,12 +16,13 @@ en:
   learning_objectives: Learning Objectives
   stepwise_step_label: Step
   references: References
+  answer_key_title: Answer Key
   eoc_chapter_review: Chapter Review
   eoc_key_terms_title: Key Terms
   eoc_summary_title: Summary
   eoc_exercises_title: Exercises
   eoc_composite_metadata_title: Chapter Review
-  eoc_answer_key_title: Answer Key
+  eoc_solutions_title: Solutions
```

Comment:
@annaludwin @wilkus27 can y'all give a polish translation for this?  

@LostMojo can you rebase to master to get the spanish translations and get @KarinaMendez2 to update them per this?

Reply:
Rebased and added!